packages/integrations/mdx/src/index.ts

```diff
@@ -62,10 +62,10 @@ function getRehypePlugins(
 	let rehypePlugins = handleExtends(mdxOptions.rehypePlugins, DEFAULT_REHYPE_PLUGINS);
 
 	if (config.markdown.syntaxHighlight === 'shiki' || config.markdown.syntaxHighlight === 'prism') {
-		rehypePlugins.push([rehypeRaw, { passThrough: nodeTypes }]);
+		rehypePlugins.unshift([rehypeRaw, { passThrough: nodeTypes }]);
```

Comment:
I was wondering if there is a slightly better way of doing this, Im never a fan of `unshift()`

```jsx
[ [rehypeRaw, { passThrough: nodeTypes}] , ...rehypePlugins ]
```
I feel this returning a new array would be easier, and a more performant solution

https://stackoverflow.com/a/67732664

Reply:
I agree, It's probably a better alternative. `Array#unshift` was just the first thing that came to mind when I thought about this issue.